docs/content/docs/deep/container-and-component-loading.md

```diff
@@ -25,225 +25,295 @@ as well as the finished diagram in [Appendix 1](#appendix-1).
-before the it is accepted. I don't go into further details about it here.
+before the it is accepted.
+I don't go into further details about it here.
+
+The `ContainerRuntime` is code written by the author of the Container, with all the logic that drives the functionality
+of the Container.
 
 :::
 
 There are a few different ways that the `Container` will get this `"code"` value:
 
-1. In the *Create New* flow this `"code"` value needs to be proposed by the Hosting Application. Once the value is
-   accepted by everyone connected (only you, the current client, in this case) the `Container` will get the event and
-   have the value.
-2. In the *Create from Existing* flow there are two scenarios.
+1. In the *Create New* flow this `"code"` value needs to be proposed by the Hosting Application.
+   Once the value is accepted by everyone connected (in this case only you, the current client) the `Container` will
+   get the event and have the value.
+1. In the *Create from Existing* flow there are two scenarios.
     1. In the *load from Summary flow* the `"code"` value is written into the Summary itself.
-    2. In the *load from op stream* flow (no Summary) the `"code"` value will be played as an op.
-
-In any case, once the `Container` has the `"code"` value it asks the `CodeLoader` to resolve the proposed code **(4)**. Since
-the Loader Package does not know anything about the `ContainerRuntime`, or Components, it needs someone to tell it
-where that code lives. This is the responsibility of the `CodeLoader`. The `CodeLoader` can dynamically pull this code
-from some source (CDN) or in some cases the code already exists on the page. Either way the `CodeLoader` needs to
-include the code on the page and return a pointer to that code to the `Container`. In the Browser, this pointer
-is an entry point to a webpacked bundle that is usually on the `window` object. In Node.js, it's a pointer to a package.
+    1. In the *load from op stream* flow (no Summary) the `"code"` value will be played as an op.
 
-![Image 4](/images/container-and-component-loading-4.jpg)
+In any case, once the `Container` has the `"code"` value it needs someone to resolve the proposed code, i.e. get it
+from wherever it lives and provide it to the `Container`.
+The Loader Package itself does not know how to do this, so it passes that responsibility to the `CodeLoader` it received
+from the Hosting Application.
+The `Container` thus ends up asking the `CodeLoader` for the `"code"`.  **(4)**
 
-At this point the `Container` has a pointer to the `ContainerRuntime` code and it uses that code to create a new
-`ContainerContext` **(5)** and executes the `instantiateRuntime` **(6.1)** on the webpack bundle with the `ContainerContext`.
+The `CodeLoader` needs to get the code and return a pointer to it to the `Container`.
+In the browser, this pointer is usually an entry point to a webpacked bundle that exists on the `window` object.
```

Comment:
```suggestion
In the browser, this pointer is usually an entry point to a JavaScript bundle that exists on the `window` object.
```

Since I don't think we want to be prescriptive in terms of bundlers.